configuration/forms.md

```diff
@@ -73,7 +73,8 @@ We currently support the following data types:
  - `"integer"`: a whole number
  - `"string"`: any collection of characters
  - `"date"`: a date in the format `YYYY-mm-dd`, for example "2019-01-28"
- - `"boolean"`: true or false, represented by the digit `1` and `0` respectively.
+ - `"boolean"`: true or false, represented by the digit `1` and `0` respectively if sent via SMS
```

Comment:
Can we update the code to map the integer to a boolean in the JSON submission case to make the documentation and implementation as simple as possible?

Reply:
no probs